boptimol/initializer.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Optional
+import logging
+
+import numpy as np
+from boptimol import Molecule
+
+logger = logging.getLogger(__name__)
+
+# Making initial guesses for the molecule (Active Learning)
+def initial_guess(mol: Molecule, init_steps: int, out_dir: str) -> np.ndarray:
+    
+    ''' Return initial coordinates guess based on the molecule (Acti)
+    Args:
+        mol: A Molecule Class Object
+        init_step: No of initial guesses to be returned
+    Returns:
+        Initial Guess Coordinates to be tried
+    '''
+    
+    start_coords = mol.parameters
+    start_energy = mol.energy(start_coords)
+    
+    init_guesses = np.random.normal(start_coords, 0.1, size=(init_steps, len(start_coords))) #TODO
```

Comment:
This might be good to separate bonds and angles and dihedrals already. I know I wrote this, but +/- 0.1 Å for a bond isn't a big deal. 0.1 radian is ~6° which is fairly large for angles.. maybe 0.05 for angles and dihedrals?